pldoc/static/js/edx-icons.js

```diff
@@ -1 +1 @@
-require('font-awesome-loader');
\ No newline at end of file
+require('font-awesome-loader');
```

Comment:
Nitpick: there should be a new line at the bottom of every file (edX code convention).